services/account/controllers/AccountController.ts

```diff
@@ -12,27 +12,15 @@ export class AccountController {
   }
 
   async getAccount(filter) {
-    try {
```

Comment:
Why take off the try/catch block? If we are taking it off the try/catch block, then we are either assuming the **filter** parameter is an object or there are further checks in the `this.repository.find` function to validate the parsed filter is an object.